R/simulateGLV.R

```diff
@@ -53,7 +56,7 @@
 #'
 #' @export
 setGeneric("simulateGLV", signature = "n.species",
-    function(n.species, A, tend = 1000, x = runif(n.species),
+    function(n.species, A, t.start = 0, t.end = 1000, t.step = 0.1, t.store = 1000, x = runif(n.species),
```

Comment:
We can specify these parameters with "..." . Since the function tDyn will be also exported, we can call it in this way. I will make it and update the file